scripts/sync.sh

```diff
@@ -242,7 +244,7 @@ function install_dependencies_on_ci {
 
 # pushes the committed changes from the local Git client to GitHub
 function push_changes {
-	git push origin HEAD:master
+	git push
```

Comment:
This must have led to synced files ending up in the wrong branch before in repos that don't use `master` as the default branch, for example https://github.com/ory/dockertest.